quran/quran.com-frontend-next
# Fix quran word copying issue

### Summary

TSIA

### How to test
- select some verses in reading mode or in translation mode, with text uthamni or indopak. You should be able to copy and get correct result. if you copy bismillah, you should get this بِسۡمِ ٱللَّهِ ٱلرَّحۡمَٰنِ ٱلرَّحِيمِ instead of بِسۡمِٱللَّهِٱلرَّحۡمَٰنِٱلرَّحِيمِ

### Testing
- tested on mobile safari 
- chrome on mac
- safari mac


### Known bugs
- It doesn't work with madaniv1 and v2 yet. We can discuss the approach for this.
